code/src/components/ThoughsOutput.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import { formatDistance } from 'date-fns';
+import { LikeButton } from './LikeButton';
+
+export const ThoughtsOutput = (props) => {
+  return (
+    props.map((thought) => {
+      return (
+        <div className="outputBox" key={thought._id}>
+          <p className="thoughtOutputText">{thought.message}</p>
+          <div className="outputBottom">
+            <LikeButton id={thought._id} hearts={thought.hearts} />
+            <div className="date-display">
+              {formatDistance(new Date(thought.createdAt), Date.now(), { addSuffix: true })}
+            </div>
+          </div>
+        </div>
+      )
+    })
+  )
+}
```

Comment:
the console error suggests the date is undefined, maybe try to console log the error and see what you get from that since I cant really think why there would be an issue with the date value being passed into formatDistance when someone does not enter the right number of characters in the happy thoughts input.

Reply:
or see above I wrote this comment before I looked at your button

Reply:
Not sure what you mean here? I cant see anything wrong with the time stamp...? It works as intended and displays how long its been since its been published

Reply:
Sorry maybe i didnt explain well or I was to abstract. Yes the time stamp works well thats sort of what i was saying I am not sure why there is an error in the console referring to this line. So I think you can fix the problem with the page when someone types less than 5 characters or more than 140 and see what happens with the console output. Then maybe the error for the timestamp is gone.